chains/orchestrator-relays/runtime/starlight/src/bridge_to_ethereum_config.rs

```diff
@@ -0,0 +1,449 @@
+    }
+}
+
+impl pallet_outbound_message_commitment_recorder::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+}
+
+// https://github.com/paritytech/polkadot-sdk/blob/2ae79be8e028a995b850621ee55f46c041eceefe/cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/bridge_to_ethereum_config.rs#L105
+impl snowbridge_pallet_outbound_queue::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type Hashing = Keccak256;
+    type AggregateMessageOrigin = AggregateMessageOrigin;
+    type GetAggregateMessageOrigin = GetAggregateMessageOrigin;
+    type MessageQueue = MessageQueue;
+    type Decimals = ConstU8<12>;
+    type MaxMessagePayloadSize = ConstU32<2048>;
+    type MaxMessagesPerBlock = ConstU32<32>;
+    type GasMeter = snowbridge_core::outbound::ConstantGasMeter;
+    type Balance = Balance;
+    type WeightToFee = WeightToFee;
+    type WeightInfo = crate::weights::snowbridge_pallet_outbound_queue::SubstrateWeight<Runtime>;
+    type PricingParameters = EthereumSystem;
+    type Channels = EthereumSystem;
+    type OnNewCommitment = CommitmentRecorder;
+}
+
+// Very stupid, but benchmarks are written assuming a fork eopch,
+// and test vectors assuming another one
+// We need allow dead code here because for regular builds this variable is not used
+// This variable is only used in test, fast-runtime or runtime-benchmarks
+#[cfg(not(feature = "runtime-benchmarks"))]
+#[allow(dead_code)]
+pub const ELECTRA_TEST_FORK_EPOCH: u64 = 0;
+#[cfg(feature = "runtime-benchmarks")]
+pub const ELECTRA_TEST_FORK_EPOCH: u64 = 80000000000;
+
+// For tests, benchmarks and fast-runtime configurations we use the mocked fork versions
+#[cfg(any(
+    feature = "std",
+    feature = "fast-runtime",
+    feature = "runtime-benchmarks",
+    test
+))]
+parameter_types! {
+    pub const ChainForkVersions: ForkVersions = ForkVersions {
+        genesis: Fork {
+            version: [0, 0, 0, 0], // 0x00000000
+            epoch: 0,
+        },
+        altair: Fork {
+            version: [1, 0, 0, 0], // 0x01000000
+            epoch: 0,
+        },
+        bellatrix: Fork {
+            version: [2, 0, 0, 0], // 0x02000000
+            epoch: 0,
+        },
+        capella: Fork {
+            version: [3, 0, 0, 0], // 0x03000000
+            epoch: 0,
+        },
+        deneb: Fork {
+            version: [4, 0, 0, 0], // 0x04000000
+            epoch: 0,
+        },
+        electra: Fork {
+            version: [5, 0, 0, 0], // 0x05000000
+            epoch:
+            ELECTRA_TEST_FORK_EPOCH,
+        }
+    };
+}
+
+// Sepolia: https://github.com/eth-clients/sepolia
+// Fork versions: https://github.com/eth-clients/sepolia/blob/main/metadata/config.yaml
+#[cfg(not(any(
+    feature = "std",
+    feature = "fast-runtime",
+    feature = "runtime-benchmarks",
+    test
+)))]
+parameter_types! {
+    pub const ChainForkVersions: ForkVersions = ForkVersions {
+        genesis: Fork {
```

Comment:
these are likely wrong now, we should put those from mainnet
look at the url above

Reply:
fixed

Reply:
BTW, @girazoki , do we need 2 configs here? Because both pointing to the mainnet now, but there is a comment about benchmarking, not sure if we break benchmarking, if we specify correct epoch for electra fork.

Reply:
yes it's correct (but please fix the comment, it says sepolia still)

for benchmarks or tests they will use the mocked ones, and for the rest the prod ones

Reply:
ACtually no this is wrong. What I meant is that you should put these ones instead of the sepolia ones: https://github.com/eth-clients/mainnet/blob/main/metadata/config.yaml